### ISSUE TITLE
Should this repository be renamed and `README.md` updated?

### ISSUE THREAD

**Issue by krassowski** [jupyter/governance#184]
Recently two well-intentioned attempts at addressing project-wide challenges:
- https://github.com/jupyterlab/team-compass/issues/212
- https://github.com/jupyterlab/team-compass/issues/213

ended up moved/replicated in this (`jupyter/governance`) repository after their scope was clarified/expanded as project-wide.

Both would have probably benefited from engaging more stakeholders early:
- during project-wide online event, e.g. currently [discontinued community call](https://github.com/jupyterlab/team-compass/issues/207)
- by ability to easily broadcast a non-binding, low-level initiative to all sub-projects (council of councils and beyond)

I am sure (and I believe everyone agrees) that both discussions were started in jupyterlab team compass not because they were aiming to exclude other subprojects but because:
- (a) it is easier to start a conversation with the group you know rather than the group you don't know
- (b) it is not obvious that we should discuss such things in this "governance" repository

Even after clarifications in https://github.com/jupyter/governance/issues/171#issuecomment-1615249925, this repository readme still says:

> The purpose of this repository is to formalize the governance process for Project Jupyter

Yes, the decision on forming a working group belongs in EC-level, and yes the decision on embracing a new a chat tool is governance-level discussion, but before we get to this stage I _think_ that in both cases authors proposing these changes were seeking initial feedback to understand who would be interested in respective changes for two reasons:
  - to define the scope - should it be jupyterlab only or extend beyond
  - to understand if there is a broader interest at all and if the effort is worth pursuing

While I cannot comment for @andrii-i @ericsnekbytes who opened the issues in question, this was certainly what was behind my decision to initially (re-)raise the topic of a new communication channel in a [jupyterlab call](https://github.com/jupyterlab/team-compass/issues/205#issuecomment-1710401793) rather than in this repository (as of today called `governance`).

I would suggest:

- [ ] renaming this repository to clarify that this is the place designated for project-wide discussion on topics that:
    a) have project-wide impact and are not related to technical aspects of Jupyter (code/protocols)
    b) may need to have EC blessing at the end of the day even if they do not need to engage actively at the early stages of the discussion
  Here are some ideas:
   - `team-compass` (this is the pattern used by JupyterLab, Notebook, JupyterHub, jupyter-server; all of these hold their internal "governance" docs in their team compass); it would be still different from EC compass
   - `governance-enhancement-proposals` - would go line-in-line with `jupyter-enhancement-proposals` which is usually about technical changes, and requires approval of SSC rather than EC.
- [ ] updating readme to reflect the above
- [ ] updating the `team-compass`es of sub-projects to add a note when opening an issue that idea that could have a project-wide impact may be better discussed here
- [ ] considering if there could be a further support for reviving community calls (or a similar effort)

CC @jupyter/executive-council

**Comment by ericsnekbytes** [comment]
> I am sure (and I believe everyone agrees) that both discussions were started in jupyterlab team compass not because they were aiming to exclude other subprojects but because:
> 
> * (a) it is easier to start a conversation with the group you know rather than the group you don't know
> * (b) it is not obvious that we should discuss such things in this "governance" repository
>
> ...
>
> ...I _think_ that in both cases authors proposing these changes were seeking initial feedback to understand who would be interested in respective changes for two reasons:
> 
> * to understand if there is a broader interest at all and if the effort is worth pursuing

I can confirm these 3 points above (as an author of one of the proposals mentioned), this post is pretty much spot on. I support the broad goals of this proposal.

>   Here are some ideas:
>   
>   * `team-compass` (this is the pattern used by JupyterLab, Notebook, JupyterHub, jupyter-server; all of these hold their internal "governance" docs in their team compass); it would be still different from EC compass
> * [ ]  updating readme to reflect the above
> * [ ]  updating the `team-compass`es of sub-projects to add a note when opening an issue that idea that could have a project-wide impact may be better discussed here
> * [ ]  considering if there could be a further support for reviving community calls (or a similar effort)

`team-compass` I think is too generic and invites confusion with other compasses. I would suggest something like...

- `jupyter-project-meta`
- `jupyter-meta-discussions`
- `jupyter-meta-team-compass`

...to communicate the idea you mentioned above: that this is a place for project wide discussions/actions to happen.

Can you clarify the purpose/focus of the meeting/community call you mentioned?

**Comment by willingc** [comment]
In the short term (and hopefully longer-term), I created `jupyter/docs` repo. See https://github.com/jupyterlab/team-compass/issues/212#issuecomment-1741590110 for further discussion.

**Comment by willingc** [comment]
I completely agree with @ericsnekbytes and @krassowski that having common repos for **across the orgs** discussion would be helpful.

**Comment by krassowski** [comment]
I guess there is no interest

**Comment by ericsnekbytes** [comment]
I'm interested and can volunteer to coordinate changes if we're in support of reopening this.

**Comment by choldgraf** [comment]
I think it makes sense to provide project-wide guidance on where to open issues that don't have a clear "home" in a sub-project. `governance/` is being used as a de-facto version of this, but that may not be the best place, and more importantly, it's not "officially" the right place for that kind of conversation, so we can't expect others to pay attention to it as if it were.

I'd suggest we create a new issue that is more to-the-point, like:

- Define a source of truth for where project-wide issues and issues without a clear home should go

Then we could either repurpose this repo or create a new one. The important thing is that we define the right place for this kind of thing. To me "documenting the place even if it's not a great place" is better than "not documenting any place because we can't decide on the perfect place"

Does that make sense?

**Comment by krassowski** [comment]
Thank you @choldgraf.

> governance/ is being used as a de-facto version of this

As is https://github.com/jupyter-governance/ec-team-compass/issues and other repos

> I'd suggest we create a new issue that is more to-the-point and focused on an MVP,

I agree. The reason this issue is not to-the-point is because I felt I needed a problem statement as a justification to open it.

> and make sure that it gets resolved

I will not have time to make sire it gets resolved it as I did not have time to chase this one so I will let someone else take a lead.

**Comment by choldgraf** [comment]
> The reason this issue is not to-the-point is because I felt I needed a problem statement as a justification to open it.

Sorry if that came across as critical of this issue - it wasn't meant to be.

> I will not have time to make sure it gets resolved it as I did not have time to chase this one so I will let someone else take a lead.

This shouldn't be your job. I'm happy to take the lead on getting something resolved as it feels like a duty of the EC to clarify this kind of thing.

**Comment by choldgraf** [comment]
Here's a follow-up issue, would you be willing to discuss with me and others there?

- https://github.com/jupyter/governance/issues/264

I also wanted to note that @ivanov opened up a related issue that also feels straightforward and actionable:

- https://github.com/jupyter/governance/issues/263
